lib/screen/main/main_screen.dart

```diff
@@ -12,6 +19,63 @@ class MainScreen extends ConsumerStatefulWidget {
 class _MainScreenState extends ConsumerState<MainScreen> {
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(body: MainHeader());
+    return Scaffold(
+        body: Row(
+      children: [
+        Expanded(
+          flex: 7,
```

Comment:
By default it's `1`, So this one we can define as `2`, It will be `2:1`
```suggestion
          flex: 2,
```